### PR TITLE
Detector adversary and record ID ability

### PR DESCRIPTION
**Pull request by ajanett-mitre**

### CODE REVIEW

File: data/adversaries/7e422753-ad7a-4401-bc8b-b12a28e69c25.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 id: 7e422753-ad7a-4401-bc8b-b12a28e69c25
-name: Incident responder
-description: A basic incident responder profile
-phases:
-  1:
-    - 3b4640bc-eacb-407a-a997-105e39788781
-  2:
-    - 02fb7fa9-8886-4330-9e65-fa7bb1bc5271
\ No newline at end of file
+        name: Incident responder
```
Comment (wbooth) [inline data/adversaries/7e422753-ad7a-4401-bc8b-b12a28e69c25.yml]: looks like some odd spacing here. let's make it consistent

File: data/abilities/detection/788ad640-d9ec-452b-8a75-31b9361f15b4.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+- id: 788ad640-d9ec-452b-8a75-31b9361f15b4
+  name: Find Windows Record IDs
+  description: Given a PID, find the associated Windows Record IDs
+  tactic: detection
+  technique:
+    attack_id: x
+    name: x
```
Comment (wbooth) [inline data/abilities/detection/788ad640-d9ec-452b-8a75-31b9361f15b4.yml]: this is fine for now but we should brainstorm ways of improving this for the response abilities

File: data/abilities/detection/788ad640-d9ec-452b-8a75-31b9361f15b4.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  repeatable: yes
+  platforms:
+    windows:
+      psh:
+        command: |
+          .\\Get-Record-IDs.ps1 $PID
```
Comment (wbooth) [inline data/abilities/detection/788ad640-d9ec-452b-8a75-31b9361f15b4.yml]: we should integrate with caldera vars for now #{host.process.id}, etc

File: data/abilities/detection/788ad640-d9ec-452b-8a75-31b9361f15b4.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+        payload:
```
Comment (wbooth) [inline data/abilities/detection/788ad640-d9ec-452b-8a75-31b9361f15b4.yml]: we'll also want to parse the result

File: data/abilities/detection/788ad640-d9ec-452b-8a75-31b9361f15b4.yml (hunk 2 above)
Comment (ajanett-mitre) [reply data/abilities/detection/788ad640-d9ec-452b-8a75-31b9361f15b4.yml]: Do you mean improving the description, tactic, technique? Or something else?

File: data/adversaries/7e422753-ad7a-4401-bc8b-b12a28e69c25.yml (hunk 1 above)
Comment (ajanett-mitre) [reply data/adversaries/7e422753-ad7a-4401-bc8b-b12a28e69c25.yml]: my bad, didn't mean to commit this change

**Pull request merged**